admin/src/auth.js

```diff
@@ -111,8 +111,33 @@ class Auth {
             .catch(() => {
                 showError("<h2>Inloggningen misslyckades</h2>Kunde inte kommunicera med servern.");
             });
+
+    }
+
+    update_member_info(member_id) {
+        fetch(config.apiBasePath + "/member/send_updated_member_info",
+            {
+                method: "POST",
+                body: JSON.stringify({ member_id: member_id }),
+                headers: { 'Content-Type': 'application/json; charset=UTF-8' },
+            })
+
+            .then(response => response.json().then(responseData => ({ response, responseData })))
+            .then(({ response }) => {
+                if (response.ok) {
+                    showSuccess("Ett email har skickats till användaren mer information om uppdateringen");
+                }
+                else {
+                    showError("<h2>Misslyckades med att skicka email till användaren om uppdaterad medlemsinformation:<br><br>" + response.status + " " + response.statusText);
```

Comment:
Can this branch happen? Won't !response.ok reject the promise?
I'm not 100% sure, but it seems reasonable to check?